functions/utils/application.js

```diff
@@ -0,0 +1,69 @@
+const createAppObject = (body) => {
+  const appObj = {
+    title: body["title"],
+    email: body["email"],
```

Comment:
Would it make sense to set a default status here, so checkApp would know that the application exists? Otherwise it'll return `"No Document"`.

Reply:
This should be done with Alper's PR to this endpoint which should be worked on this sprint!